catalogue/webapp/services/wellcome/catalogue/filters.ts

```diff
@@ -1,13 +1,9 @@
 import { palette } from '@weco/catalogue/components/PaletteColorPicker';
 import {
-  CatalogueResultsList,
-  Work,
-  Image,
+  ImageAggregations,
+  WorkAggregations,
 } from '@weco/catalogue/services/wellcome/catalogue/types';
-import {
-  Article,
-  ContentResultsList,
-} from '@weco/catalogue/services/wellcome/content/types/api';
+import { ArticleAggregations } from '@weco/catalogue/services/wellcome/content/types/api';
```

Comment:
Could be imported from `@weco/catalogue/services/wellcome/catalogue/types` (looks like it was the OG intention too)

Reply:
Done!